README.md

```diff
@@ -354,90 +354,132 @@ settings that machine might have that could differ from the machine used for wri
 #### Build & Run the image
 
 OpenCfp provides a ready-to-use docker-compose.yml file.
-If you need to add configuration options specific to your working environment (network ...) add them to a docker-compose.override.yml file.
-Instead of updating the *ENV_PROD*.yaml.dist to set your environment, with the docker file, you only have to modify the environment variable in the docker-compose.yml.
 
+Instead of updating the *ENV_PROD*.yaml.dist file to set your environment, with the docker file, you only have to modify the environment variables in the docker-compose.yml.
```

Comment:
Change this part because we don't modify the `docker-compose.yml` file but we overload it with the `docker-compose.override.yml` file.

Also change the example below please

Reply:
Is now changed for a really short explication for the docker-compose.override.yml